pkg/controller/bastion/actuator_reconcile.go

```diff
@@ -291,12 +298,31 @@ func ensureSecurityGroupRules(openstackClientFactory openstackclient.Factory, op
 		return errors.New("shootSecurityGroups must not be empty")
 	}
 
-	rules := []rules.CreateOpts{IngressAllowSSH(opt, opt.EtherType, secGroupID, opt.EtherIpAddress), EgressAllowSSHToWorker(opt, secGroupID, shootSecurityGroups[0].ID)}
-	for _, item := range rules {
-		if err := createSecurityGroupRuleIfNotExist(openstackClientFactory, item); err != nil {
-			return err
+	// remove all exist ingress allow ssh rules for patch purpose
+	ingressSSHRules, err := getRulebyName(openstackClientFactory, ingressAllowSSHResourceName(opt.BastionInstanceName))
+	if openstackclient.IgnoreNotFoundError(err) != nil {
+		return err
+	}
+
+	if len(ingressSSHRules) != 0 {
+		for _, rule := range ingressSSHRules {
+			err = deleteRule(openstackClientFactory, rule.ID)
+			if err != nil {
+				return fmt.Errorf("failed to delete security group rule %s-%s", rule.Description, rule.ID)
+			}
 		}
 	}
+
+	// create ingress rules
+	for _, etherItem := range ethers {
+		rules := []rules.CreateOpts{IngressAllowSSH(opt, etherItem.EtherType, secGroupID, strings.Join(etherItem.CIDRs, ","))}
```

Comment:
`strings.Join(etherItem.CIDRs, ",")`
can you pass multiple cidrs, joined by `,` to `RemoteIPPrefix`? Or do you instead have to create multiple rules instead? In the UI this is not allowed (but must not mean anything) and in the API documentation it was not mentioned that this is possible.
```go
rules.CreateOpts{
	RemoteIPPrefix: cidr,
}
```

Reply:
committed in eff38149